tests/contrib/django/django_tests.py

```diff
@@ -200,6 +201,25 @@ def test_user_info(self):
         self.assertTrue('email' in user_info)
         self.assertEquals(user_info['email'], 'admin@example.com')
 
+    def test_user_info_raises_operational_error(self):
+        user = User(username='admin', email='admin@example.com')
+        user.set_password('admin')
+        user.save()
+
+        self.assertTrue(
```

Comment:
I'll keep it in the same style as the rest of the file for now. We can do a sweeping/global change later.